recipes/fscacher/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fscacher" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a560253b5d6f8c00c1e1ff7b48731d59ce091544a13d4284c43c40b38ff6ffd
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

Reply:
No, the package requires Python 3.6 or higher.

Reply:
For `noarch: python` packages you can add `- python >=3.6` in `host` and `run`. The bot will complain that it's missing as soon as this suggestion is applied.

Reply:
Done.